content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,197 @@
+  are tainted with spot=true:NoSchedule.
+- To strike a balance between cost and resource availability, we want jobs to use up to 1000 cores of on-demand nodes, then use up to 2000 cores 
+  of spot nodes.
+
+As batch admins, we define two ResourceFlavors that represent the two types of nodes
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: ondemand
+  labels:
+    instance-type: ondemand 
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: spot
+  labels:
+    instance-type: spot
+taints:
+- effect: NoSchedule
+  key: spot
+  value: "true"
+```
+
+Then we define the quotas by creating a ClusterQueue as follows
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ClusterQueue
+metadata:
+  name: research-pool
+spec:
+  namespaceSelector: {}
+  resources:
+  - name: "cpu"
+    flavors:
+    - name: ondemand
+      quota:
+        min: 1000
+    - name: spot
+      quota:
+        min: 2000
+```
+
+Note that the order of flavors matters, Kueue will attempt to fit jobs in the available quotas according to the order unless the job has an
+explicit affinity to specific flavors. 
+
+For each namespace, we define a LocalQueue that points to the ClusterQueue above.
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: LocalQueue
+metadata:
+  name: training
+  namespace: team-ml
+spec:
+  clusterQueue: research-pool
+```
+
+Admins create the above setup once. As batch users, we are able to find the queues we are allowed to submit to by listing the `LocalQueues` 
+in our namespace. The command is similar to the following: `kubectl get -n my-namespace localqueues`
+
+To submit a job, we simply set the `kueue.x-k8s.io/queue-name` annotation as follows:
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sample-job-
+  annotations:
+    kueue.x-k8s.io/queue-name: main
+spec:
+  parallelism: 3
+  completions: 3
+  template:
+    spec:
+      tolerations:
+      - key: spot
+        operator: "Exists"
+        effect: "NoSchedule"
+      containers:
+      - name: dummy-job
+        image: gcr.io/k8s-staging-perf-tests/sleep:latest
+        args: ["30s"]
+        resources:
+          requests:
+            cpu: 1
+      restartPolicy: Never
+```
+
+
+Kueue operates by forcing the job to be created in suspended mode. Once the job is at the head of the ClusterQueue, Kueue evaluates if it can start
+by checking if the resources requested by the job fit the available quota. 
+
+In the above example, the job tolerates Spot resources, and so if there are previously admitted jobs consuming all existing on-demand quota but
+not all of Spot’s, Kueue admits the job using the Spot quota. Kueue does this by issuing a single update to the job object that: 
+- Changes the suspended flag to false 
+- Adds the nodeSelector term instance-type: spot to the pod template so that when the pods are created by the job controller, they can only schedule 
```

Comment:
```suggestion
- Adds the nodeSelector term `instance-type: spot` to the pod template so that when the pods are created by the job controller, they can only schedule 
```